src/app/modules/map-config/services/map-layer-form-builder/models.ts

```diff
@@ -20,6 +20,7 @@ export enum GEOMETRY_TYPE {
   fill = 'fill',
   line = 'line',
   circle = 'circle',
+  circleHeat = 'circle-heatmap',
   heatmap = 'heatmap',
   label = 'label'
```

Comment:
Why do we use 'label' in the form for the geometry types, even though we transform it to 'symbol' to fill the configuration ? Wouldn't it make things easier to just use 'symbol' and skip the conversion from 'label' to 'symbol' in multiple places ?
@MohamedHamouGisaia @sebbousquet @mbarbet